asciidoctorj-core/src/main/resources/specifications/.jrubydir

```diff
@@ -0,0 +1,11 @@
+..
+.
+asciidoctor-1.5.2.gemspec
+coderay-1.1.0.gemspec
+erubis-2.7.0.gemspec
+haml-4.0.5.gemspec
+open-uri-cached-0.0.5.gemspec
+slim-2.0.3.gemspec
+temple-0.6.10.gemspec
+thread_safe-0.3.4-java.gemspec
+tilt-2.0.1.gemspec
```

Comment:
Personally I'd prefer to generate this file from the build script.
In particular without a complete OSGI integration test suite it will be very probable that we miss one name in the future.